### PR TITLE
hide social previews if not premium

### LINKED ISSUE #69: Yoast SEO social previews are blank without premium
When the premium version of Yoast is not active, the social previews are empty. This appears to be related to defining the globals rather than being a style override (e.g. `display: none`).

### PR DESCRIPTION
**Pull request by jazzsequence**
This PR does some cleanup of Yoast loading code -- it's actually not in our best interest to spoof Yoast Premium because it changes the labels to say that Yoast SEO Premium is active. 

It adds a helper function that is used here and can be used later (e.g. with #59) to determine if Yoast Premium is active, and then uses that function to load styles that conditionally hide the social previews if Premium is not active.

Fixes #69

### CODE REVIEW

File: inc/namespace.php [hunk 1]
```diff
@@ -248,3 +253,36 @@ function override_wizard_styles() {
...
+		return;
+	}
+
+	/**
+	 * This targets the 6th and 7th components panel in the Yoast
+	 * sidebar, which corresponds to the Facebook and Twitter social
+	 * preview buttons. If Yoast ever adds more panels to this sidebar,
+	 * this will need to be updated.
+	 */
+	$styles = 'div.components-panel div:nth-child(6n) div.yoast.components-panel__body, div.components-panel div:nth-child(7n) div.yoast.components-panel__body {
```
Comment (roborourke) [inline inc/namespace.php:274]: A good future update might be to ask if they can provide hooks to switch these on/off, or at least provide CSS class names for these items.

**Review by roborourke — APPROVED** [review]

**Comment by roborourke** [comment]
If someone does install Yoast premium will our installer overrides place it in `vendor` or does it have a different package name?

**Comment by jazzsequence** [comment]
> If someone does install Yoast premium will our installer overrides place it in vendor or does it have a different package name?

It should be `wordpress-seo-premium` so it would install separately.

File: inc/namespace.php (hunk 1 above)
Comment (jazzsequence) [reply inc/namespace.php:274]: Yeah, this feels super hacky. The other option was disabling the inner elements that _do_ have classes, but the `border-bottom` is on the wrapping container, so you ended up with an extra thick border under the Google preview button.

**Pull request merged**